benchmarks/inference/mii/src/utils.py

```diff
@@ -123,22 +141,34 @@ def parse_args(
     # Common args
     parser = argparse.ArgumentParser(parents=parents)
     parser.add_argument(
-        "--model", type=str, default="meta-llama/Llama-2-7b-hf", help="Model name"
+        "--model", type=str, default=None, help="HuggingFace.co model name"
     )
     parser.add_argument(
         "--deployment_name",
         type=str,
-        default="mii-benchmark-deployment",
-        help="Deployment name for MII server",
+        default=None,
+        help="When using FastGen backend, specifies which model deployment to use",
```

Comment:
Should we update this description since the `deployment_name` is used in `call_aml` as well?